rotkehlchen/chain/evm/transactions.py

```diff
@@ -428,6 +430,48 @@ def _get_erc20_transfers_for_ranges(
                 queried_ranges=[(start_ts, end_ts)],
             )
 
+    def _query_and_save_erc20_transfers_for_range(
+            self,
+            address: ChecksumEvmAddress,
+            period: TimestampOrBlockRange,
+            location_string: str,
+            update_ranges: bool = True,
+    ) -> None:
+        """Helper function to abstract ERC20 transfer querying functionality for different range types"""  # noqa: E501
```

Comment:
same for docstring for update ranges. (perhaps it will end up repeating but not sure what's best)